src/test/java/bio/terra/common/TestUtils.java

```diff
@@ -261,7 +261,7 @@ public static <T> T mapFromJson(String content, Class<T> valueType) throws IOExc
       return objectMapper.readValue(content, valueType);
     } catch (Exception ex) {
       logger.error(
-          "unable to map JSON response to " + valueType.getName() + "JSON: " + content, ex);
+          "unable to map JSON response to " + valueType.getName() + " JSON: " + content, ex);
       throw ex;
```

Comment:
I took this suggestion since it was small, but to keep myself honest will share that this fell [outside of the scope](https://mtlynch.io/human-code-reviews-2/#respect-the-scope-of-the-review) of my changeset.  I have a hard time cutting myself off sometimes :)